bedrock/security/templates/security/third-party-injection-policy.html

```diff
@@ -0,0 +1,216 @@
+    degrade the user experience. When third-party software is found to
+    destabilize the browser, Firefox may intervene.
+  </p>
+
+  <p>Interventions* may include, but are not limited to:</p>
+
+  <ol class="mzp-u-list-styled">
+    <li>
+      Detecting instability locally, notifying the user, and offering them
+      the opportunity to block specific modules from being injected.
+    </li>
+    <li>
+      Detecting instability locally, automatically blocking the unstable
+      module, and providing the user an option to unblock it.
+    </li>
+    <li>
+      Detecting aggregate instability across many users, and proactively
+      blocking the module globally across all Firefox instances.
+    </li>
+  </ol>
+
+  <p>
+    <em>
+      *Current versions of Firefox do not automatically block problematic
+      modules or prompt the user to do so, but we are exploring this
+      functionality. Firefox does maintain a block list which is already
+      used when problems are discovered from crash reporting and bug
+      reports. See below for more information.
+    </em>
+  </p>
+
+  <p>
+    If a third-party feature achieved with Firefox injection works well,
+    that’s great. However, if we become aware of a problem, we will
+    contact the vendor if they have shared a responsive support contact
+    with Mozilla. We are happy to alert external vendors about such
+    issues and welcome proactive communication with vendors before
+    issues arise. Please contact us at
+    <a href="mailto:thirdpartysoftware@mozilla.com">thirdpartysoftware@mozilla.com</a>.
+  </p>
+
+  <p>
+    While we prefer to work cooperatively with other developers, our
+    resources are finite and oftentimes we have to act quickly. We reserve
+    the right to block third-party injections whenever we believe this is
+    in the best interest of Firefox users.
+  </p>
+
+  <p>
+    We intend to review this position over time and may be more restrictive
+    with third-party module injection in the future.
+  </p>
+
+  <h2>In-Browser Diagnostics</h2>
+
+  <p>
+    When we encounter an interoperability issue caused by a third-party module,
+    we look for a workaround or implement a mitigation patch in Firefox while
+    contacting the vendor to request a fix. If those attempts are not
+    successful, we globally block the module using our
+    <a href="https://wiki.mozilla.org/Blocklisting/DLL">blocklist</a>.
+    In most cases, blocking can be limited to the problematic versions of the
+    module and modules can be unblocked after a fix is implemented. Not all
+    third-party interoperability issues are addressed. For example, some may
+    only affect a small number of users.
+  </p>
+
+  <p>
+    In Firefox, we provide an in-browser diagnostics page. The about:third-party
+    page (only available on Firefox for Windows and explained
+    <a href="https://support.mozilla.org/kb/identify-problems-third-party-modules-firefox-windows">here</a>)
```

Comment:
**issue:** screenreader users will see link text "here" out of context

**suggestion:** add sr-only text or aria label to provide context ("identify Firefox for Windows third party problems" or similar)